tests/api/test_security.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import unittest
+import os
+
+
+class Testsecurity(object):
+    def setup(self):
+        # app['TESTING'] = True
+        # self.app = app.test_client()
+        pass
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
```

Comment:
or alternatively i can check if user is in mongoengine or not, but the initial idea was just to  test wrong  email and password to see if login works properly